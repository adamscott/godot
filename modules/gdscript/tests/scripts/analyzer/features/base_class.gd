const external_script = preload("gdscript_to_preload.notest.gd")

func virtual_method(_ext: external_script) -> String:
	return "Base"

func test():
	var external_instance = external_script.new()
	print("virtual_method call: " + virtual_method(external_instance))
