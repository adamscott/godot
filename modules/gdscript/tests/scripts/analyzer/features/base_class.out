GDTEST_OK
virtual_method call: Base
